src/modules/Popup/Popup.js

```diff
@@ -0,0 +1,270 @@
+import React, { Component, PropTypes } from 'react'
+import cx from 'classnames'
+import _ from 'lodash'
+import { META } from '../../lib'
+import Portal from '../../addons/Portal/Portal'
+import PopupContent from './PopupContent'
+import PopupHeader from './PopupHeader'
+
+const _meta = {
+  name: 'Popup',
+  type: META.TYPES.MODULE,
+}
+
+
+/**
+ * A Popup displays additional information on top of a page.
+ */
+export default class Popup extends Component {
+  static propTypes = {
+    /** Display the popup without the pointing arrow */
+    basic: PropTypes.bool,
+
+    /** You may pass a content as children of the Popup */
+    children: PropTypes.node,
+
+    /** Classes to add to the Accordion className. */
+    className: PropTypes.string,
+
+    /** Simple text content for the popover */
+    content: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+
+    /** A Flowing popup have no maximum width and continue to flow to fit its content */
+    flowing: PropTypes.bool,
+
+    /** Takes up the entire width of its offset container */
+    fluid: PropTypes.bool,
+
+    /** Header displayed above the content in bold */
+    header: PropTypes.string,
+
+    /** Whether the popup should not close on hover */
+    hoverable: PropTypes.bool,
+
+    /** Invert the colors of the popup */
+    inverted: PropTypes.bool,
+
+    /** The node where the popup should mount.. */
+    hideOnScroll: PropTypes.bool,
+
+    /** Horizontal offset in pixels to be applied to the popup */
+    offset: PropTypes.number,
+
+    /** Event triggering the popup */
+    on: PropTypes.oneOf(['hover', 'click', 'focus']),
+
+    /** Positionning for the popover */
+    position: PropTypes.string,
```

Comment:
Since there are fixed positions supported, let's also define them in `_meta.props` and use an enum here:

```js
_meta = {
  props: {
    positioning: [
      'top left',
      'top center',
      'top right',
      'right center',
      'bottom right',
      'bottom center',
      'bottom left',
      'left center',
    ]
  },
}

positioning: PropTypes.oneOf(_meta.props.positioning),
```